web/ui/static/js/graph.js

```diff
@@ -518,7 +519,6 @@ Prometheus.Graph.prototype.transformData = function(json) {
       color: palette.color()
     };
   });
-  Rickshaw.Series.zeroFill(data);
```

Comment:
It seems like from the test that it fills all steps: https://github.com/shutterstock/rickshaw/blob/eb1bf7c9c0cfcd38134a9a92a486fe176d1b191c/tests/Rickshaw.Series.js#L137-L153